scripts/set_s3_creds

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
```

Comment:
We frequently copy data from s3 using awscli or s3cmd using temporary credentials.  To avoid everyone having to deal with writing their own scripts to accomplish the same thing, I've written this one for all to use.